src/main/java/uk/gov/hmcts/reform/juddata/camel/util/MappingConstants.java

```diff
@@ -3,13 +3,26 @@
 import java.sql.Timestamp;
 import java.util.Date;
 
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+
 public class MappingConstants {
```

Comment:
why constants class created inside util package?
I am not sure will constant fill contains some methods also

Reply:
yes you can add file with default method that is new Java 8 Interface with default method.

Reply:
why package structure looks different then ..could you please move it constants package or rename the class name ..